File: MODUL-3/SOAL 4/PRAK304-2410817120017-RIKAFAULIANARAHMI.py
a = int(input("Bilangan = "))
if a==0:
    print("Nol")
elif a>=1 and a<=9:
    print("Satuan")
elif a== 10 or a>=20 and a<=99:
    print("Puluhan")
elif a>=11 and a<=19:
    print("Belasan")
else :
    print("Anda Menginput Melebihi Limit Bilangan")

File: MODUL-3/SOAL 4/PRAK304-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

int main () {
    int a;
    printf("Masukan Bilangan : ");
    scanf("%d" ,&a);
    if (a==0){
        printf("Nol");
    }
    else if (a>=1 && a<=9){
        printf("Satuan");
    }
    else if (a==10 || a>=20 && a<=99){
        printf("Puluhan");
    }
    else if (a>=11 && a<=19){
        printf("Belasan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilanngan");
    }
}